mifospay/src/main/res/values/styles_frame.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="FrameLayout"/>
+
+    <style name="FrameLayout.Base">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">match_parent</item>
+    </style>
+
+    <style name="FrameLayout.Container" parent="FrameLayout.Base">
```

Comment:
Instead of '<style name="FrameLayout.Container" parent="FrameLayout.Base">' it would be much better if you do 
'<style name="FrameLayout.Base.Container" >'